lib/cylc/scheduler.py

```diff
@@ -1069,7 +1069,7 @@ def _load_suite_params(self, row_idx, row):
             elif self.options.startcp is None:
                 self.options.startcp = value
                 LOG.info('+ start point = %s' % value)
-        elif key == 'fcp':
+        elif key == ['fcp', 'final_point']:
```

Comment:
should that be `key in`?

Reply:
Yes. Just realised the mistake. Updated in the latest change.

Reply:
Ooh, good spot!